src/configure/mod.rs

```diff
@@ -525,17 +534,26 @@ fn get_offline_autoconfig(context: &Context, param: &LoginParam) -> Option<Login
 
 fn try_imap_connections(
```

Comment:
The relationship between try_smtp_connections and try_imap_connection could be reworked, too (e.g. if it was autoconfig the `param.mail_*.split_at(at)` (cutting away the @domain part of the addr) will not be reached.  I guess thats on purpose but it takes you a long time to realize. 

But then it seemed too risky again because a bug here may easily stay unnoticed but will keep away new users. Maybe I should just put some comments.